blogposts/2021/java-scala-kotlin-code-intelligence.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Precise code intelligence for Java, Scala, and Kotlin'
+description: 'We are excited to announce that Sourcegraph now supports precise code intelligence for Java, Scala, and Kotlin.'
+author: Ólafur Páll Geirsson
+publishDate: 2021-12-17T00:00-07:00
+tags: [blog]
+slug: java-scala-kotlin-code-intelligence
+heroImage: //images.ctfassets.net/le3mxztn6yoo/t4Qpcq5kA0AYM24Ws4mOk/4edf5502a936bbec90c262fa00355aed/sourcegraph-mark.png
+socialImage: https://about.sourcegraph.com/blog/blogIndexing1200x627.jpg
```

Comment:
I copy-pasted this image from another blog post but we should probably have something else. Would it make sense to use the cross-repo example instead?

Reply:
I think that would make sense!